cmd/lncli/cmd_open_channel.go

```diff
@@ -117,6 +117,16 @@ var openChannelCommand = cli.Command{
 				"channel. This must not be set at the same " +
 				"time as local_amt",
 		},
+		cli.StringFlag{
+			Name: "utxos",
```

Comment:
I think it's possible to actually have it accept a slice/array instead? So then a _bit_ less parsing needed on our end. 

So it could be `--utxo=Y --utxo=K`, etc.

Reply:
maybe take inspiration from `lncli wallet fund --inputs ...` they use a json array